marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/mapping/getUris.sjs

```diff
@@ -0,0 +1,21 @@
+/**
+ Copyright (c) 2020 MarkLogic Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+'use strict';
+const core = require('/data-hub/5/artifacts/core.sjs')
+
+xdmp.securityAssert("http://marklogic.com/data-hub/privileges/read-mapping", "execute");
+
+var stepName, limit;
```

Comment:
Is "limit" required then? I think to make this safe, it should default to something like 10 if no value is provided.

Reply:
"limit" is required parameter. Should we still have a default ?

Reply:
It's defined as a string, which means null could be passed in. It should really be defined as an integer. DS will likely generate an Integer for it, in which case it can still be null. But yeah, if it's an Integer, then it seems fine for it to be required.